parts/k8s/kubernetesmastervars.t

```diff
@@ -88,7 +88,7 @@
     "apiVersionKeyVault": "2018-02-14",
     "apiVersionNetwork": "2018-08-01",
     "apiVersionManagedIdentity": "2015-08-31-preview",
-    "apiVersionAuthorization": "2018-01-01-preview",
+    "apiVersionAuthorization": "2018-09-01-preview",
```

Comment:
Is it possible to use a stable ARM schema instead of a preview?

Reply:
Unfortunately the issue (principal not found) fix is available only in this right now. When they come up with stable version we can move to that.